src/n2o.erl

```diff
@@ -31,7 +31,7 @@ start(_,_) -> catch n2o_vnode:load([]), X = supervisor:start_link({local,n2o},n2
                 X.
 
 start_mqtt_ring() ->
-  [ n2o_pi:start(#pi{module=n2o_vnode,table=mqtt,sup=n2o,state=[],name=Pos})
+  [ n2o_pi:start(#pi{module=n2o_vnode,table=ring,sup=n2o,state=[],name=Pos})
```

Comment:
The commit is not clear, this commit overrides the current.
You need separate branch https://github.com/synrc/n2o/tree/v6.5